models/response.js

```diff
@@ -0,0 +1,56 @@
+var User = require('./user');
+var Service = require('./request');
```

Comment:
These won't be necessary since we will only be concerned with `Response` associations in this file